recipes/python/python_3.5.1.oe.sig

```diff
@@ -0,0 +1,2 @@
+b7f832e6d84b406db4c854e3eb46411e6931bc98  Python-3.5.1.tgz
+8e4202008ed03191b6e492dcf3448aeff1f70db9  v3.5.1.tar.gz
```

Comment:
Is both of these really needed?  It looks like the last signature is not needed/used in the current recipe.